src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Utility.cs

```diff
@@ -1407,6 +1867,20 @@ private static bool VectorContainsNonAsciiChar(Vector128<byte> asciiVector)
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static bool VectorContainsNonAsciiChar(Vector256<byte> asciiVector)
```

Comment:
Do we need wrapper methods for this simple case?

Is there any reason it can't just be `asciiVector != Vector256<byte>.Zero` instead which allows `ptest; jcc` rather than `movmsk, cmp; jcc`?

Reply:
I was doing this for code reuse/readability.

Would you prefer to just inline the function?

Reply:
I think inlining it manually in this case would be better.

The abstractions like this are generally helpful when we have more complex logic differing between platforms or when there is a high likelihood to need to identify and change the pattern for the same pattern repeatedly in the future.

In cases like this, where we're really just doing a trivial comparison check, I don't think the helper buys us much in terms of readability/maintainability and it does have drawbacks in the form of forcing the JIT to do more work to inline/optimize the code.